docs/_private_integrations/build-a-private-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Build a private integration using the Zapier Platform
+order: 2
+layout: post-toc
+redirect_from: /private_integrations/
+---
+
+# Build a private integration using the Zapier Platform
+
+This guide will walk you through what steps you need to take to build a [private integration](https://platform.zapier.com/private_integrations/private-vs-public-integrations) from start to finish. We will include information for if you are building with either [Zapier Platform UI or Zapier Platform CLI](https://platform.zapier.com/docs/vs). 
+
+> **Tip**: If you haven’t used Zapier before, learn the basics in our [Zapier Getting Started Guide](https://zapier.com/learn/zapier-quick-start-guide/).
+
+## Prerequisite
+
+* You’ll need a [Zapier account](https://zapier.com/).
```

Comment:
Should this link to the [sign up](https://zapier.com/sign-up) page directly?

Reply:
Good shout, I'll change that now!